api/server/image.go

```diff
@@ -112,7 +113,7 @@ func (s *Server) postImagesCreate(ctx context.Context, w http.ResponseWriter, r
 			OutStream:   output,
 		}
 
-		err = s.daemon.Repositories().Pull(image, tag, imagePullConfig)
+		err = s.daemon.Repositories().Pull(image, tag, imagePullConfig, dryRun != "")
```

Comment:
@aaronlehmann Seems that the CLI does not send the "dryRun=" parameter when set to false.
Do you see a possible improvement here though?